sample_project/Assets/SampleViewer/Samples/Geocoding/Geocoder.cs

```diff
@@ -53,26 +54,42 @@ public class Geocoder : MonoBehaviour
 
     private InputActions inputActions;
     private bool isLeftShiftPressed;
+    private TouchControls touchControls;
 
     private void Awake()
     {
+#if !UNITY_IOS && !UNITY_ANDROID && !UNITY_VISIONOS
         inputActions = new InputActions();
+#else
+        touchControls = new TouchControls();
+#endif
     }
 
     private void OnEnable()
     {
+#if !UNITY_IOS && !UNITY_ANDROID && !UNITY_VISIONOS
```

Comment:
Feels like its written backwards. Why not do `if UNITY_IOS || UNITY_ANDROID || UNITY_VISIONOS` for touch behavior and put the normal behavior in the else statement?

Reply:
because this is checking to see if it should enable touch controls or normal controls. If it isn't iOS, VisionOS, or Android it does the normal controls and not the touch controls. Else it does touch controls. Inverting the macro will not change anything